superset-frontend/src/utils/dates.ts

```diff
@@ -16,6 +16,18 @@
  * specific language governing permissions and limitations
  * under the License.
  */
+
+import 'dayjs/locale/de';
+import 'dayjs/locale/es';
+import 'dayjs/locale/fr';
+import 'dayjs/locale/it';
+import 'dayjs/locale/ja';
+import 'dayjs/locale/ko';
+import 'dayjs/locale/pt';
+import 'dayjs/locale/pt-br';
+import 'dayjs/locale/ru';
+import 'dayjs/locale/zh';
```

Comment:
This is a bit of a problem because now we are making localization dependent on this file. Is there a better way to achieve this?

Reply:
This is fixed now, we are dynamically including locales from translations directory.